src/modules/proto.haml

```diff
@@ -73,10 +69,13 @@
                         <!-- Other modules. -->
                         </map>
                         ```
-
                     * A team can have a minimum required size with the `min=""` attribute.
                     * A players gamemode can be changed by using a the `<game-mode>adventure</game-mode>` kit.
-
+                    * New world border module, that can be used to create a shifting and/or auto resizing world border.
```

Comment:
It is already listing some none-breaking changes, a separate list or labeling the changes that are breaking and moving them to the top of the change list would work.